### ISSUE TITLE
p-calendar month picker doesn't display correct year

### ISSUE THREAD

**Issue by DerStimmler** [primefaces/primeng#11076]
**I'm submitting a ...**
```
[x] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Plunkr Case (Bug Reports)**

https://stackblitz.com/edit/github-rgjy7j?file=src/app/app.component.html
https://github-rgjy7j.stackblitz.io

**Current behavior**
When you have a month picker and click on the arrows to switch the current year, the label of the year doesn't change.

**Expected behavior**
The label of the year should change when switching the year.

**Minimal reproduction of the problem with instructions**
Just add a p-calendar component with the view attribute set to "month".
`<p-calendar view="month" dateFormat="mm/yy" [readonlyInput]="true"></p-calendar>`

**What is the motivation / use case for changing the behavior?**
It should work properly.

**Please tell us about your environment:** Windows, Linux

* **Angular version:** 12.X

* **PrimeNG version:** 13.0.4

* **Browser:** all
 
* **Language:** all

* **Node (for AoT issues):** `node --version` =   14.15.0

**Comment by DerStimmler** [comment]
The bug exists also in the PrimeNG documentation: https://www.primefaces.org/primeng/showcase/#/calendar

**Comment by DerStimmler** [comment]
Same as #11024 .
Is fixed in v13.1.0
